crates/node/p2p/src/gossip/driver.rs

```diff
@@ -128,34 +130,40 @@ impl GossipDriver {
                 message_id: id,
                 message,
             } => {
-                trace!(target: "p2p::gossip::driver", "Received message with topic: {}", message.topic);
+                debug!("Received message with topic: {}", message.topic);
                 if self.handler.topics().contains(&message.topic) {
-                    debug!(target: "p2p::gossip::driver", "Handling message with topic: {}", message.topic);
                     let status = self.handler.handle(message);
-                    debug!(target: "p2p::gossip::driver", "Reporting message validation result: {:?}", status);
                     _ = self
                         .swarm
                         .behaviour_mut()
                         .gossipsub
                         .report_message_validation_result(&id, &src, status);
                 }
             }
+            libp2p::gossipsub::Event::Subscribed { peer_id, topic } => {
+                debug!(target: "p2p::gossip::driver", "Peer: {:?} subscribed to topic: {:?}", peer_id, topic);
```

Comment:
Bit of a mixture of logs with manual `target` settings, prefer to go one way or the other. Whole codebase has favored manually setting the `target` to date, but I don't really mind going either direction for net-new components like this so long as it's consistent.

Reply:
Right - i'm trying to move away from this because it's already handled by tracing and most of the time we're just making the code less legible. I also think it's fine to have a mix of both in the case where we do want to specify the target manually.

Reply:
That's fine. I'm not too attached, definitely don't need to go either way in this PR.

Eventually I'd like to follow up on a global prefix, which probably means removing the `target` filters across the codebase. Makes using the `EnvFilter` a _lot_ easier.